lib/rest-api.php

```diff
@@ -10,47 +10,6 @@
 	die( 'Silence is golden.' );
 }
 
-/**
- * Registers the REST API routes needed by the Gutenberg editor.
- *
- * @since 2.8.0
- */
-function gutenberg_register_rest_routes() {
```

Comment:
While these functions are "internal" to the plugin, nonetheless it may be prudent to put them on a deprecation schedule similar to what's done in #13412 , and otherwise remove the implementation of the function to be a noop ?

Reply:
Removing them would be a noop, because the routes are already registered in WP 5.0. As long as WP 5.0 is the minimum requirement, it's duplicate code.

Reply:
I mean specifically the function `gutenberg_register_rest_routes`, rather than removing outright, deprecating for some time, in the off-chance someone would have been calling it directly.

Reply:
Oh. Your call. Personally, I don't think anyone _should've_ been calling it directly.